Makefile-az.mk

```diff
@@ -232,6 +232,9 @@ az-rmnodeclaims-fin: ## Remove Karpenter finalizer from all nodeclaims (use with
 az-rmnodeclaims: ## kubectl delete all nodeclaims; don't wait for finalizers (use with care!)
 	kubectl delete --wait=false nodeclaims --all
 
+az-taintnodes: ## Run e2etests
```

Comment:
The comment is wrong

Reply:
Thx for the catch. Got a little eager on the completion. Another reason for a follow up